packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -264,6 +264,30 @@ void main() {
       expect(picker.initialDateTime, isNotNull);
     });
 
+    testWidgets('initial date honors minuteInterval', (WidgetTester tester) async {
+        DateTime newDateTime;
+        await tester.pumpWidget(
+          CupertinoApp(
+            home: SizedBox(
+              width: 400,
+              height: 400,
+              child: CupertinoDatePicker(
+                onDateTimeChanged: (DateTime d) => newDateTime = d,
+                initialDateTime: DateTime(2018, 10, 10, 10, 3),
+                minuteInterval: 3,
+              )
+            )
+          )
+        );
+
+        // 03 -> 06
```

Comment:
This might be clear now but might no longer be clear in a year. Write a full sentence here.